aggregator/priceNotifier.go

```diff
@@ -0,0 +1,168 @@
+package aggregator
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+)
+
+const epsilon = 0.0001
+
+// ArgsPriceNotifier is the argument DTO for the price notifier
+type ArgsPriceNotifier struct {
+	Pairs   []*ArgsPair
+	Fetcher PriceFetcher
+	Notifee PriceNotifee
+}
+
+// ArgsPair is the argument DTO for a pair
+type ArgsPair struct {
+	Base                      string
+	Quote                     string
+	PercentDifferenceToNotify uint32
+	TrimPrecision             float64
```

Comment:
What is the advantage of using the trimPrecision?

Reply:
simpler mathematics. Otherwise I would have to compute powers of ten which are heavier to execute rather than float operations.